doc/articles/uno-builds-troubleshooting.md

```diff
@@ -64,3 +64,11 @@ To generate a binlog file from the command line:
   ```
 
 - Once the build has finished, a file named `msbuild.binlog` is generated next to the `.csproj` file.
+
+## Common Build errors troubleshooting
+
+### iOS: Verification of iOS environment is running. Please try again in a moment
```

Comment:
I am wondering if the iOS troubleshooting page we should link to this current build error page you are editing here
https://github.com/unoplatform/uno/blob/master/doc/articles/common-issues-ioscatalyst.md/#L1
![image](https://github.com/user-attachments/assets/cf4bc82b-0551-4eb8-afe8-d2928a2dc776)